homeassistant/components/august.py

```diff
@@ -218,17 +223,25 @@ def get_doorbell_detail(self, doorbell_id):
     def _update_doorbells(self):
         detail_by_id = {}
 
+        _LOGGER.debug("Start retrieving doorbell details.")
         for doorbell in self._doorbells:
             detail_by_id[doorbell.device_id] = self._api.get_doorbell_detail(
                 self._access_token, doorbell.device_id)
 
+        _LOGGER.debug("Completed retrieving doorbell details.")
+
         self._doorbell_detail_by_id = detail_by_id
 
     def get_lock_status(self, lock_id):
         """Return lock status."""
         self._update_locks()
         return self._lock_status_by_id.get(lock_id)
 
+    def get_lock_door_status(self, lock_id):
+        """Return door lock status."""
```

Comment:
Personally, I would make this comment (and the one on line 236 a little more descriptive, it's unclear to me what the difference is between `lock status` and `door lock status`.

Reply:
Updated comments for both to better explain the difference. Please check if this explains it better (I know what I'm saying <g>).